WikiWatershed/model-my-watershed
# Feature/lol/scenario dropdown

Connects #168 

This PR adds real functionality to the dropdowns and creates abstracted modals that can be used anywhere.

To test:

1. Draw an area and enter modeling mode.
2. Click the "+" twice to create two new scenarios.
3. Use the dropdown to rename a scenario.
4. Use the dropdown to delete a scenario, you should be confronted with a modal confirmation box.
5. Try sharing and printing (similar functionality).
6. Attempt to create two scenarios with the same name. Try to trick the UI to create scenarios with the same slug by making names like "-A-" and "+A+".
